engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -141,12 +143,19 @@ public List<User> getUsers(String accountId, String orgId, boolean adminsOnly) {
 
         // Micrometer doesn't like when tags are null and throws a NPE.
         String accountIdTag = accountId == null ? "" : accountId;
-        getUsersTotalTimer.stop(meterRegistry.timer("rbac.get-users.total", "accountId", accountIdTag, "users", String.valueOf(users.size())));
-        rbacUsers.set(users.size());
+        getUsersTotalTimer.stop(meterRegistry.timer("rbac.get-users.total", "accountId", accountIdTag));
+        getRbacUsersGauge(accountIdTag).set(users.size());
 
         return users;
     }
 
+    private AtomicInteger getRbacUsersGauge(String accountIdTag) {
+        return rbacUsers.computeIfAbsent(accountIdTag, /* accountIdTag */ key -> {
+            Set<Tag> tags = Set.of(Tag.of("accountId", key));
+            return meterRegistry.gauge("rbac.users", tags, new AtomicInteger());
+        });
```

Comment:
Isn't it better to call it `accountIdTag` instead of `key`? or even `accountId` ? 

```suggestion
        return rbacUsers.computeIfAbsent(accountIdTag, accountId -> {
            Set<Tag> tags = Set.of(Tag.of("accountId", accountId));
            return meterRegistry.gauge("rbac.users", tags, new AtomicInteger());
        });
```

Reply:
We can't repeat `accountIdTag` since it's already taken but `accountId` will work, yes.

Reply:
yep, any is fine :D